ansible/roles/supervisor/tasks/main.yml

```diff
@@ -4,6 +4,7 @@
   shell: 'which supervisord'
   register: which_supervisord
   changed_when: false
+  failed_when: which_supervisord.rc != 0 and which_supervisord.rc != 1
```

Comment:
I'm not sure I understand why this should fail if `rs != 1`. Doesn't an rc of 1 mean that it failed to find it?